teachbooks/cli/main.py

```diff
@@ -24,11 +21,12 @@ def build(ctx, path_source, publish, release, process_only):
     """Pre-process book contents and run Jupyter Book build command"""
     from teachbooks.release import make_publish
     from jupyter_book.cli.main import build as jupyter_book_build
+    import os
```

Comment:
Shouldn't `os` be a dependency in the toml file if we use it here?

Reply:
That's an unused import that isn't used anymore. I used it previously to test some stuff, but now they're not needed anymore and therefore deleted. Thank you for pointing it out!